New package: DuckStudio.FufuTools version 1.3.1-1

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Related #156080 but didn't solve it.

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

> [!IMPORTANT]
>
> Since @DuckDuckStudio create the related issue, and he/she is the publisher & author of the package. This pull request should also ask for approval from him/she.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156134)

Dragon1573 committed:
New package: DuckDuckStudio.Fufu_Tools version 1.3.1-1

Signed-off-by: 是蓝兔哟～ <49941141+Dragon1573@users.noreply.github.com>

DuckDuckStudio commented on manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.installer.yaml:
```diff
@@ -0,0 +1,18 @@
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2024-06-01
```
```suggestion
ReleaseDate: 2024-05-29
```
这个应该是指发布日期,,,吧？

Dragon1573 committed:
Apply suggestions from code review

- Correction of release date

Co-authored-by: 鸭鸭「カモ」 <Yzcbs123@163.com>

Dragon1573 committed:
Try switching architecture of the package

- Check if the installer wizard is 64-bit only.

DuckDuckStudio commented:
在旧PR中有提到静默安装参数

Dragon1573 commented on manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.installer.yaml:
```diff
@@ -0,0 +1,18 @@
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
```
```suggestion
- silentWithProgress
InstallerSwitches:
  Silent: /verysilent /suppressmsgboxes
  SilentWithProgress: /verysilent /suppressmsgboxes
```

Dragon1573 committed:
Apply suggestions from code review

- Add installer switches for silent mode

Dragon1573 commented:
> Dragon1573,
> 
> The check-in policies require a moderator to approve PRs from the community. Our [moderators](https://github.com/microsoft/winget-pkgs/issues/15674#issue-908889039) are community volunteers, please be patient and allow them sufficient time to review your submission.

Hooray! @DuckDuckStudio 🎉 It seems your main application was built for `x64` only, but you packaged them into a `x86` installer wizard. So _Komac_ detected the architecture of wizard and also mark the whole package as `x86` instead of `x64`.

-----

好耶！看上去你的主程序是仅针对 `x64` 架构进行构建输出的，但你用打包工具把它制作成了一个 `x86` 架构的安装向导文件。 _Komac_ 检测到了安装向导的架构，于是也把你的主程序标记为 `x86` 架构了。

DuckDuckStudio commented on manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.locale.zh-CN.yaml:
```diff
@@ -0,0 +1,21 @@
+PackageVersion: 1.3.1-1
+PackageLocale: zh-CN
+Publisher: 鸭鸭工作室
+PublisherUrl: https://github.com/DuckDuckStudio
```
```suggestion
PublisherUrl: https://duckduckstudio.github.io/yazicbs.github.io/
```

DuckDuckStudio commented on manifests/d/DuckDuckStudio/Fufu_Tools/1.3.1-1/DuckDuckStudio.Fufu_Tools.locale.zh-CN.yaml:
```diff
@@ -0,0 +1,21 @@
+PublisherUrl: https://github.com/DuckDuckStudio
+Author: 鸭鸭「カモ」
+PackageName: 芙芙工具箱
+PackageUrl: https://github.com/DuckDuckStudio/Fufu_Tools
```
```suggestion
PackageUrl: https://duckduckstudio.github.io/yazicbs.github.io/Tools/Fufu_Tools/
```

DuckDuckStudio commented:
And I have a request, could you please put this program in `DuckStudio` folder instead of `DuckDuckStudio` folder?
If possible, please also change the ID to `DuckStudio.FufuTools`.
Thank you.

---

如果可以，请将`DuckDuckStudio` 改为 `DuckStudio`，并修改ID。
感谢。

Dragon1573 committed:
Apply suggestions from code review

- Change package URL from GitHub Repo link to publisher's site hosted on GitHub Pages
- Change publisher URL from GitHub Profile page to publisher's own blog site hosted on GitHub Pages

Co-authored-by: 鸭鸭「カモ」 <Yzcbs123@163.com>

Dragon1573 committed:
Change package identifier: DuckDuckStudio.Fufu_Tools ==> DuckStudio.FufuTools

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

DuckDuckStudio commented:
Why it failing again....

Dragon1573 commented:
@stephengillie - Could you please manually trigger the _Azure Pipelines_ again? 🙏🏼 The manifest can pass the `winget validate`, and able to install in `amd64` Windows Sandbox instance.

stephengillie commented:
Automatic Validation ended with:

> Installation failed with exit code -1978334964
> 2024-06-02 01:32:10.146 [CLI ] ShellExecute installer failed: 5
>                           An error occurred while trying to read the source file:

(Automated response - build 882.)

stephengillie commented:
Manual Validation ended with:
> 
> 2024-06-03 16:43:46.714 [FAIL] Installer failed security check. Url: https://github.com/DuckDuckStudio/Fufu_Tools/releases/download/v1.3.1-1/Fufu_Tools_Setup.v1.3.1-1.exe Result: 0x80004005

(Automated response - build 882.)

Dragon1573 commented:
@DuckDuckStudio - Seems there're security risks of the package and it's detected by _Manual Validation_ process.

Really wierd ... 😕

stephengillie commented:
@wingetbot run

Dragon1573 commented:
> [!NOTE]
>
> The related issue has been resolved by another pull request. This PR can be closed as rejected.
